cli/modules.c

```diff
@@ -56,6 +56,8 @@ const char* getObjName(uint32_t id) {
   #define get_cwd getcwd
 #endif
 
+#include "thirdparty/argparse/argparse.h"
```

Comment:
`argparse` isn't a module of pocketlang (ie you cannot import argparse in pocketlang). So don't include this here. As for now include this in `main.c`